js/wizard/wizardTabGeneric.js

```diff
@@ -34,12 +34,28 @@ OCA = OCA || {};
 		 */
 		isToggling: false,
 
+		/**
+		 * @property {bool} - indicates whether form changes have been made
+		 */
+		hasUnsavedChanges: false,
+
+		/**
+		 * @property {bool} - list of form elements that have changes
+		 */
+		saveQueue: [],
+
 		/** @inheritdoc */
 		init: function(tabIndex, tabID) {
+			var self = this;
 			this.tabIndex = tabIndex;
 			this.tabID = tabID;
 			this.spinner = $('.ldapSpinner').first().clone().removeClass('hidden');
+			this._disableButton('.ldap_action_continue, .ldap_action_save');
 			_.bindAll(this, '_toggleRawFilterMode', '_toggleRawFilterModeConfirmation');
+			$('.ldap_action_save').click(function(event) {
```

Comment:
I guess we don't have a narrower scope for the selectors ? (sadly this isn't a backbone view, else we could use this.$('.ldap_action_save')'